ocaml-lsp-server/src/custom_requests/req_documentation.mli

```diff
@@ -0,0 +1,38 @@
+val meth : string
+
+val capability : string * [> `Bool of bool ]
+
```

Comment:
Are you sure that it is necessary to exports every submodules? I think that only `Request_params` is mandatory.